test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,86 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+	it('Consume GET Service', async () => {
+		const response = await axios.get('https://httpbin.org/ip').catch((error) => {
+			console.log(error.toJSON());
+		});
+
+		expect(response.status).to.equal(StatusCodes.OK);
+		expect(response.data).to.have.property('origin');
+	});
+
+	it('Consume GET Service with query parameters', async () => {
+		const query = {
+			name: 'John',
+			age: '31',
+			city: 'New York',
+		};
+
+		const response = await axios
+			.get('https://httpbin.org/get', { query })
+			.catch((error) => {
+				console.log(error.toJSON());
+			});
+
+		expect(response.status).to.equal(StatusCodes.OK);
+		expect(response.config.query).to.eql(query);
+	});
+
+	it('Consume HEAD Service', async () => {
+		const response = await axios
+			.head('https://httpbin.org/get')
+			.catch((error) => {
+				console.log(error.toJSON());
+			});
+
+		expect(response.status).to.equal(StatusCodes.OK);
+		expect(response.data).to.be.empty;
+	});
+
+	it('Consume HEAD Service', async () => {
+		const response = await axios
+			.head('https://httpbin.org/get')
+			.catch((error) => {
+				console.log(error.toJSON());
+			});
+
+		expect(response.status).to.equal(StatusCodes.OK);
+		expect(response.data).to.be.empty;
```

Comment:
The url for HEAD service is https://httpbin.org/#/operations/Request%20inspection/get_headers
![image](https://user-images.githubusercontent.com/36433884/175042479-5b1d9dc8-1040-4fef-8cf8-d1c2a9139f9a.png)